index.js

```diff
@@ -257,10 +260,12 @@ function get20s(/*Your Code Here*/){
  
  For example, if removeArtist is invoked with the artists array and the number 0, it will remove Amedeo Modigliani from our dataset and return the number 19. */
 
-function removeArtist(/*Your Code Here*/){
-   /*Your Code Here*/
+function removeArtist(array, number){
```

Comment:
The second function parameter is the index number of the item to be removed.  So it can not be used as the delete count in the splice method.  

This line is incorrect:    artists.splice(0,number);

This is the syntax for splice:  splice(start, deleteCount)